src/models/strengthExercise.ts

```diff
@@ -14,6 +14,8 @@ export enum StrengthExerciseCategory {
 }
 
 export enum StrengthExerciseMovement {
+  Unilateral = 'unilateral',
+  Bilateral = 'bilateral',
   Isolation = 'isolation',
   Compound = 'compound'
 }
```

Comment:
I don't think this makes sense. Since we've basically removed support for the column that contains isolation, we should create a new enum type for it and mark it as deprecated.